Dockerfile

```diff
@@ -17,9 +17,9 @@ RUN set -uex; \
     # gpg: key 5072E1F5: public key "MySQL Release Engineering <mysql-build@oss.oracle.com>" imported
     key='A4A9406876FCBD3C456770C88C718D3B5072E1F5'; \
     export GNUPGHOME="$(mktemp -d)"; \
-    (gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" \
-    || gpg --batch --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$key" \
-    || gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key"); \
+    (gpg --batch --keyserver keyserver.ubuntu.com --recv-keys "$key" \
+    || gpg --batch --keyserver keys.openpgp.org --recv-keys "$key" \
+    || gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key"); \
```

Comment:
build actually times out on this one using `hkp` scheme, but I feel it's important to keep it in there for reference